specification.md

```diff
@@ -357,6 +383,17 @@ enum ServerCapabilities {
 }
 ```
 
+#### agent_identification
```

Comment:
I think that "new_instance_uid" would be too specific field and wanted something with a bit broader scope so I thought about "AgentIdentification", though also wondering about "AgentIdentifcationOverride", "AgentIdentificationOffer" or so. If you had anything else in mind @tigrannajaryan I will gladly accept it

Reply:
Is there anything else that we may want to add to AgentIdentification message in the future?

Reply:
> Is there anything else that we may want to add to AgentIdentification message in the future?

I don't have anything on my mind right now though I'm actually wondering if instead creating `AgentIdentification`, a model used in `AgentDescription` could be followed (so repeated KeyValue). I don't really have a strong use case but wondering if this would give an opportunity for the server to send some attributes back (both identifying and non-identifying) that would be persisted by the agent. Not sure if it makes much sense though (since server could always attach them later itself if needed)

Reply:
We can add keyvalue list as an additional field later if needed. I think what we have for now is fine.